sdk/core/azure-core/test/ut/transport_adapter.cpp

```diff
@@ -169,98 +251,6 @@ namespace Azure { namespace Core { namespace Test {
-    std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    // 2 connections must be available at this point
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsOnPool("httpbin.org"), 2);
-
-    std::thread t3(threadRoutine);
-    std::thread t4(threadRoutine);
-    std::thread t5(threadRoutine);
-    t3.join();
-    t4.join();
-    t5.join();
-    // wait a few ms for connections to go back to pool.
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
-    // Two connections re-used plus one connection created
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsOnPool("httpbin.org"), 3);
-  }
-
-#ifdef RUN_LONG_UNIT_TESTS
-  TEST_F(TransportAdapter, ConnectionPoolCleaner)
-  {
-    std::string host("http://httpbin.org/get");
-
-    auto threadRoutine = [host]() {
-      auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, host);
-      auto response = pipeline.Send(context, request);
-      checkResponseCode(response->GetStatusCode());
-      auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
-      CheckBodyFromBuffer(*response, expectedResponseBodySize);
-    };
-
-    // one index expected from previous tests
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsIndexOnPool(), 1);
-
-    std::cout
-        << "Running Connection Pool Cleaner Test. This test takes more than 3 minutes to complete."
-        << std::endl
-        << "Add compiler option -DRUN_LONG_UNIT_TESTS=OFF when building if you want to skip this "
-           "test."
-        << std::endl;
-
-    // Wait for 100 secs to make sure any previous connection is removed by the cleaner
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000 * 100));
-
-    std::cout << "First wait time done. Validating state." << std::endl;
-
-    // index is not affected by cleaner. It does not remove index
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsIndexOnPool(), 1);
-    // cleaner should have remove connections
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsOnPool("httpbin.org"), 0);
-
-    // Let cleaner finish
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
-
-    std::thread t1(threadRoutine);
-    std::thread t2(threadRoutine);
-    t1.join();
-    t2.join();
-
-    // 2 connections must be available at this point and one index
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsIndexOnPool(), 1);
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsOnPool("httpbin.org"), 2);
-
-    // At this point, cleaner should be ON and will clean connections after on second.
-    // After 5 seconds connection pool should have been cleaned
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000 * 100));
-
-    std::cout << "Second wait time done. Validating state." << std::endl;
-
-    // EXPECT_EQ(Http::CurlSession::ConnectionsIndexOnPool(), 0);
-    EXPECT_EQ(Http::CurlConnectionPool::ConnectionsOnPool("httpbin.org"), 0);
```

Comment:
Any harm in keep this part?

Reply:
All this part was moved to be the first tests to be ran.  It's 2 tests that I am moving from being in the middle to being first.